tests/NSubstitute.Analyzers.Tests.Shared/DocumentationTests/DocumentationTests.cs

```diff
@@ -62,7 +62,9 @@ private static List<Block> GetParsedDocumentation(FileInfo fileInfo)
 
         private static string GetRulesDocumentationDirectoryPath()
         {
-            return Path.GetFullPath(Path.Combine(Path.GetDirectoryName(Assembly.GetExecutingAssembly().Location), "..\\..\\..\\..\\..\\documentation\\rules"));
+            var locations = new[] { "..", "..", "..", "..", "..", "documentation", "rules" };
+            var rulesPath = string.Join(Path.DirectorySeparatorChar, locations);
```

Comment:
I had to adjust "traversing" to parent directories so as it works correctly on Linux